PURL-TYPES.rst

```diff
@@ -301,6 +315,28 @@ maven
       pkg:maven/net.sf.jacob-projec/jacob@1.14.3?classifier=x64&type=dll
 
 
+mlflow
```

Comment:
@dbczumar would you be able to do a quick review or loop in someone else from Databricks to verify I got the various model identifiers right?

Reply:
Doing some extra tests, it looks like the Databricks implementation of MLflow is case insensitive and the Azure ML implementation is case sensitive. I did not see the MLflow spec explicitly call out expected behavior, so documenting both here.